content/algorithm-and-data-structures/index.mdx

```diff
@@ -1,8 +1,56 @@
+This page gives you a brief overiew of the most important topics in Algortihm and Datastrucure Basics. Links for further research will also be included
+
+## Sorting Algorithms
+
+Depending on the underlying Data, there are various Algorithms that can be used to arrange them in an ordered way, going from Basic 1 by 1 Sorting to advanced complex procedures.
+
+- [YouTube Playlist](https://www.youtube.com/watch?v=JU767SDMDvA&list=PL9xmBV_5YoZOZSbGAXAPIq1BeUf4j20pl) - A playlist that covers the basics of simple but/and effective Sorting Algorithms
+- [Sorting in C#](https://dotnetcoretutorials.com/2020/05/10/basic-sorting-algorithms-in-c/) - Every wondered what Array.Sort actually does in the background, this is the right page for you (also with code samples)
+- [Runtime Complexity](https://www.geeksforgeeks.org/time-complexities-of-all-sorting-algorithms/) - A Quick overview of how long you can expect your algorithm to be running. Useful when you wondere whether that 100 item List Sort should really be taking around 15 minutes to complete.
+
+## The Big-O-Notation and Landau Symbols
+
+This chapther hopefully takes out the "O No" of the "Big O Notation", lets get started.
+
+- [Brief overview](https://medium.com/swlh/an-overview-of-big-o-notation-b30d6047e601) This page breaks down the basics of what the Big-O-Notation intends to do, also check out the Link(https://www.bigocheatsheet.com/) to the Big-O-Cheatsheet
+- [Video Explanation](https://www.youtube.com/watch?v=__vX2sjlpXU) A 5 minute guide on how to use the big O Notation to calculate the worst case complexity of an algorithm.
+- [Best/Worst/Average Runtime](https://algorithmtutor.com/Analysis-of-Algorithm/Best-Average-Worst-case-analysis-of-Algorithms/) A short overview of the different runtimes that can be calculated.
+- [O,Ω,Θ = ???](https://jarednielsen.com/big-o-omega-theta/) - There is more to the Landausymbols than just the big O. Theta and Omega, the most important ones besides O, are covered in this brief read.
+
+## Datastructures
+
+There are many different ways to store data efficently, the following part will demonstrate the DOs and DONTs of Datastructures
+
+- [Linked List](https://www.geeksforgeeks.org/data-structures/linked-list/) Geeks for Geeks explains the concept of a linked list with examples in JAVA (which is quite simmilar to C# so it should be easy to translate)
+- [I am Groot?](https://algorithmtutor.com/Data-Structures/Tree/Tree-ADT/) For all those ever wondering whether they are a tree or not, this website covers all the features and terminologies about trees (I am Groot!).
+- [~~un~~balanced](http://btechsmartclass.com/data_structures/avl-trees.html) AVL Trees take normal Trees on another level in terms of searchtime and Tree height. Learn it all here!
+
+##Pathfinding
+
+Pathfinding Algorithms are very important in many different situation, ranging from Google Maps routes to HTTP Packets hopping the Internet
```

Comment:
A list would be great. They don't necessarily need an explanation what the difference is, unless you're up to it.